rtb/norme_di_progetto_ver0.5.0.typ

```diff
@@ -531,9 +540,182 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+
+*Responsabile di Progetto*\
+Il Responsabile ha il compito fondamentale di coordinare i membri del gruppo e rappresentarlo nelle comunicazine con il proponente e i committenti.\
+I suoi principali compiti sono:
+- Approvare la documentazione;
+- Gestire la pianificazione del progetto come descritta sopra;
+- Coordinare i membri controllando lo stato del backlog e assegnando le attività da portare a termine;
+- Studiare e gestire l'analisi dei rischi;
+- Gestire le comunicazioni con l'esterno.
+
+*Amministratore di Progetto*\
+L'amministratore definisce, controlla e gestisce il sistema in cui lavora il gruppo, con piena responsabilità sull'efficacia ed efficienza del Way of Working.\
+I suoi principali compiti sono:
+- Essendo il vice del Responsabile, sostituirlo nel caso di impedimenti;
+- Migliorare l'ambiente di lavoro automatizzando il più possibile i processi;
+- Gestione e ottimizzazione dei processi;
+- Redigere e mantenere la documentazione;
+- Gestire le infrastrutture e i servizi.
+
+
+*Analista*\
+L'analista è la figura più importante nelle fasi iniziali del progetto, egli infatti è responsabile dell'analisi dei requisiti, grazie alla quale si potrà passare alla vera e propria progettazione del software.\
+Ha il compito di:
+- Studiare il problema descritto dal proponente e il relativo dominio d'uso;
+- Raccogliere e studiare i bisogni dei committenti;
+- Redigere il documento Analisi dei Requisiti;
+- Studiare i requisiti definendo la loro complessità.
+
+*Progettista*\
+Il progettista determina le scelte realizzative del progetto, scegliendo i design patterns più adeguati per soddisfare i requisiti individuati dagli analisti in un'architettura che modelli il problema. Il progettista seguirà lo sviluppo particolarmente, ma non la manutenzione.\
+Ha il compito di:
+- Sviluppare un prodotto economico, facilmente manutenibile a partire dal lavoro dell'analista;
+- Favorire efficienza ed efficacia grazie alle scelte tecniche effettuate;
+- Garantire un basso grado di accoppiamento grazie ad un sistema ben strutturato.
+
+*Verificatore*\
+Il verificatore controlla il lavoro svolto dagli altri componenti del gruppo, assicurandosi che le norme vengano attuate correttamente. \
+Ha il compito di:
+- Verificare la correttezza delle attività tramite gli strumenti e tecniche definiti nelle Norme di Progetto;
+- Verificare la conformità dei prodotti ai requisiti funzionali e di qualità;
+
+*Programmatore*\
+Il programmatore è incaricato di produrre il codice necessario a sviluppare il software che risponda ai bisogni dei proponenti indivuati dagli analisti secondo l'architettura e i design patterns proposti dai progettisti.\
+Ha il compito di:
+- Implementare la Specifica Tecnica scritta dal Progettista;
+- Scrivere codice mantenibile, che rispetti le `Norme di Progetto`;
+- Creare test per la verifica e validazione del codice;
+- Redigere il manuale utente.
```

Comment:
- "software" va in corsivo
- "design patterns" va in corsivo
- "Norme di Progetto" non essendo con estensione non dovrebbe avere il monospace da quello che è detto nelle NdP, altrimenti andrebbe fatto in tutti gli altri casi e mi pare che non sia così
- "test" va in corsivo,  è stato fatto anche precedentemente